.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] Please, consider to add `test.md` file into `.gitignore` file. You can see, that actually it is in your repository and not ignored, as required.
<img width="913" alt="Screen Shot 2022-02-08 at 14 30 32" src="https://user-images.githubusercontent.com/49064106/152969572-a35790c5-cd60-42fb-93b6-9a74960addb1.png">